$OpenBSD$

Fixes this bug:

raco pkg show
Installation-wide:
open-output-file: cannot open output file
  path: /usr/local/share/racket/pkgs/.LOCKpkgs.rktd
  system error: Permission denied; errno=13

https://github.com/plt/racket/commit/39fda5ec9eac8bd17d276cc35e28c55ed66e9e64

--- collects/setup/unixstyle-install.rkt.orig	Tue Mar 17 20:23:06 2015
+++ collects/setup/unixstyle-install.rkt	Fri May 15 02:11:02 2015
@@ -506,8 +506,9 @@
     (delete-directory rktdir)))
 
 (define (skip-dot-files!)
-  (current-skip-filter ; skip all dot-names
-   (lambda (p) (regexp-match? #rx"^[.]" (basename p)))))
+  (current-skip-filter
+   ;; skip all dot-names, except ".LOCK..."
+   (lambda (p) (regexp-match? #rx"^[.](?!LOCK)" (basename p)))))
 
 (define (make-install-copytree)
   (define copytree (move/copy-tree #f))
